examples/roles/permissions-ns-without-ip-resolution.yaml

```diff
@@ -0,0 +1,33 @@
+# This example shows the roles required for a user to be able to use Mizu in all namespaces.
```

Comment:
should the description include "the without ip resolution"?

Reply:
fixed